packages/react-core/src/components/Button/Button.tsx

```diff
@@ -50,6 +50,8 @@ export interface ButtonProps extends React.HTMLProps<HTMLButtonElement> {
   icon?: React.ReactNode | null;
   /** Sets the button tabindex. */
   tabIndex?: number;
+  /** Adds small styling to a link button */
```

Comment:
I don't think this styling is specific to link buttons.  Can you update the description here.

Reply:
Sure.